hasher-matcher-actioner/webapp/src/utils/constants.jsx

```diff
@@ -37,3 +41,12 @@ export const StatNames = Object.freeze({
   HASHES: 'hashes',
   MATCHES: 'matches',
 });
+
+// Matches threatexchange.content_type.* for each subclass of ContentType, get_name()
+// >>> from threatexchange.content_type.meta import get_all_content_types
+// >>> [content_type.get_name() for content_type in get_all_content_types()]
```

Comment:
Love this however it doesn't work for me on my devcontainer. get the following error:
```AttributeError: partially initialized module 'threatexchange.signal_type.signal_base' has no attribute 'SignalType' (most likely due to a circular import)```

Reply:
Yeah, you'll need to `pip install .` again. Threatexchange needs to be updated. I fixed that in 0.0.22.